microsoft/testsuites/core/mana.py

```diff
@@ -0,0 +1,164 @@
+import re
+
+from lisa import (
+    LisaException,
+    Node,
+    SkippedException,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.operating_system import Linux, Posix
+from lisa.sut_orchestrator import AZURE
+from lisa.tools import Dmesg, KernelConfig, Modprobe
+
+
+@TestSuiteMetadata(
+    area="mana_driver",
+    category="functional",
+    description="""
+    This test suite checks for the presence of mana and mana_ib drivers
+    """,
+)
+class ManaDriverCheck(TestSuite):
+    MANA_DRIVER_MESSAGE_PATTERN = re.compile(r"\[\s*[0-9.]+\s*\]\s+mana\s+.*")
+    MANA_IB_DRIVER_MESSAGE_PATTERN = re.compile(r"\[\s*[0-9.]+\s*\]\s+mana_ib\s+.*")
```

Comment:
add the sample messages

Reply:
updated

Reply:
I don't see the update with sample messages